doop-api/aggregate_test.go

```diff
@@ -86,6 +90,7 @@ func TestAggregateOfTwoClusters(t *testing.T) {
 	//test merging of structures on all levels of the report
 	expected := mustParseJSON[doop.AggregatedReport](t, "fixtures/output-both-clusters.json")
 	actual := AggregateReports(inputSet, BuildFilterSet(url.Values{}))
+	actual.Sort()
```

Comment:
Should we move the sort into AggregateReports? I think the main function has that, too.

Reply:
We don't need sorting for the AggregateReports call in metric collection (which is probably going to have the most calls), so I'd like to leave it opt-in.